### PR TITLE
feat: support vitepress's code snippet import

### LINKED ISSUE #3459: [Feature Request] Some way to ignore certain tokens like `<<<`
VitePress provides a `<<< @/filepath` syntax to [import code snippets](https://vitepress.dev/guide/markdown#import-code-snippets). But with volar it shows `Illegal tag name. Use '&lt;' to print '<'. vue(12)`. Which is technically right, but I need some way to skip checking for this.

### PR DESCRIPTION
**Pull request by so1ve**
close #3459

### CODE REVIEW

File: packages/vue-language-core/src/plugins/file-md.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const scriptSetupReg = /\\\<[\s\S]+?\>\n?/g;
 const sfcBlockReg = /\<(script|style)\b[\s\S]*?\>([\s\S]*?)\<\/\1\>/g;
 const angleBracketReg = /\<\S*\:\S*\>/g;
 const linkReg = /\[[\s\S]*?\]\([\s\S]*?\)/g;
+const codeSnippetImportReg = /^<<<\s*.+/gm;
```
Comment (brc-dd) [inline packages/vue-language-core/src/plugins/file-md.ts:12]: `<<<` can be indented too: https://github.com/brc-dd/vitepress-whyframe-demo/blob/main/src/components/alert/index.md?plain=1#L21

Comment (so1ve) [reply packages/vue-language-core/src/plugins/file-md.ts]: Makes sense, thanks!

**Review by brc-dd — APPROVED** [review]
Tested it locally. Seems to working fine. Thanks!

**Comment by johnsoncodehk** [comment]
Thanks!

**Pull request merged**